### PR TITLE
Makes brig physicians unable to become bloodsuckers.

### PR DESCRIPTION
**Pull request by Arturlang**
## Why It's Good For The Game
This might not be something that is wanted.

## Changelog
:cl:
tweak: Bloodsuckers can no longer start as brig physicians.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/bloodsucker/bloodsucker.dm [hunk 1]
```diff
@@ -25,7 +25,7 @@
 	antag_flag = ROLE_BLOODSUCKER
 	false_report_weight = 1
 	restricted_jobs = list("AI","Cyborg")
-	protected_jobs = list("Chaplain", "Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel", "Chief Engineer", "Chief Medical Officer", "Research Director", "Quartermaster")
+	protected_jobs = list("Chaplain", "Security Officer", "Warden", "Detective", "Brig Physician", "Head of Security", "Captain", "Head of Personnel", "Chief Engineer", "Chief Medical Officer", "Research Director", "Quartermaster")
```
Comment (Akrilla) [inline code/game/gamemodes/bloodsucker/bloodsucker.dm:28]: Need to add this to be restricted from mid round and late join antags.

Comment (Trilbyspaceclone) [inline code/game/gamemodes/bloodsucker/bloodsucker.dm:28]:
```suggestion
	protected_jobs = list("Chaplain", "Security Officer", "Warden", "Detective", "Brig Physician", "Head of Security", "Captain", "Head of Personnel", "Chief Engineer", "Chief Medical Officer", "Research Director", "Quartermaster") //Skyrat change adds Brig Physician to protected jobs
```

Comment (Arturlang) [reply code/game/gamemodes/bloodsucker/bloodsucker.dm:28]: Good point, ill do that

Comment (Arturlang) [reply code/game/gamemodes/bloodsucker/bloodsucker.dm:28]: Ah yeah, i probably wont be making changes here upstream but worth doing anyways

Comment (Arturlang) [reply code/game/gamemodes/bloodsucker/bloodsucker.dm]: Dynamic ones right?

Comment (Akrilla) [reply code/game/gamemodes/bloodsucker/bloodsucker.dm]: Yup

Comment (Arturlang) [reply code/game/gamemodes/bloodsucker/bloodsucker.dm]: Did it

**Pull request merged**